pandas/core/groupby/generic.py

```diff
@@ -1304,7 +1351,7 @@ class DataFrameGroupBy(NDFrameGroupBy):
                   klass='DataFrame',
                   axis='')
     @Appender(_shared_docs['aggregate'])
-    def aggregate(self, arg, *args, **kwargs):
+    def aggregate(self, arg=None, *args, **kwargs):
```

Comment:
Does this have to change as well? We've actually run into quite a few complaints during the typing process for classes with shared ancestors or mixins having different signatures for methods so this would generate more complaints I think

Reply:
We do need this change to make `arg` optional, so that `.agg(foo=('A', 'sum'))` is handled correctly.